store/tikv/lock_resolver.go

```diff
@@ -271,7 +271,8 @@ func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc Regi
 //    commit status.
 // 3) Send `ResolveLock` cmd to the lock's region to resolve all locks belong to
 //    the same transaction.
-func (lr *LockResolver) ResolveLocks(bo *Backoffer, callerStartTS uint64, locks []*Lock) (int64, error) {
+// If noWait is true, ResolveLocks would not wait the secondary lock TTL to expire.
+func (lr *LockResolver) ResolveLocks(bo *Backoffer, callerStartTS uint64, locks []*Lock, noWait bool) (int64, error) {
```

Comment:
When should we use `noWait == false` mode like before or we just use the new mode to avoid unintended rollback operations?

Reply:
`noWait` is now removed